tank_lab_to_nwb/convert_towers_task/towerspositiondatainterface.py

```diff
@@ -74,49 +74,36 @@ def convert_data(self, nwbfile: NWBFile, metadata_dict: dict, stub_test: bool =
             nwbfile.add_epoch_column('label', 'name of epoch')
 
-            blocks = matin['log']['block']
-            epoch_start_fields = blocks[0][0]['start'][0]
-            n_epochs = len(epoch_start_fields)
-            epoch_start_dts = [date_array_to_dt(x[0]) for x in epoch_start_fields]
-            epoch_durationss = [timedelta(seconds=x[0][0]) for x in blocks[0][0]['duration'][0]]
-
-            epoch_windows = []
-            for j in range(n_epochs):
-                start = epoch_start_dts[j] - session_start_time
-                end = start + epoch_durationss[j]
-                epoch_windows.append([start.total_seconds(), end.total_seconds()])
-
-            for j, window in enumerate(epoch_windows):
-                nwbfile.add_epoch(start_time=window[0], stop_time=window[1], label='Epoch'+str(j))
-
-            trial_start_fields = []
-            trial_duration_fields = []
-            n_trials = []
-            for j in range(n_epochs):
-                trial_start_fields.append(blocks[0][0]['trial'][0][j]['start'][0])
-                trial_duration_fields.append(blocks[0][0]['trial'][0][j]['duration'][0])
-                n_trials.append(len(trial_start_fields[j]))
-            trial_starts = [y[0][0]+epoch_windows[j][0] for j, x in enumerate(trial_start_fields) for y in x]
-            trial_ends = [y1[0][0]+epoch_windows[j][0]+y2[0][0]
-                          for j, x in enumerate(zip(trial_start_fields, trial_duration_fields))
-                          for y1, y2 in zip(x[0], x[1])]
-
+            epoch_start_dts = [array_to_dt(x.start) for x in matin['log']['block']]
+            epoch_durations = [timedelta(seconds=x.duration) for x in matin['log']['block']]
+            epoch_start_nwb = [(epoch_start_dt - session_start_time).total_seconds()
+                               for epoch_start_dt in epoch_start_dts]
+            epoch_end_nwb = [(epoch_start_dt - session_start_time + epoch_duration).total_seconds()
+                             for epoch_start_dt, epoch_duration in zip(epoch_start_dts, epoch_durations)]
+            for j, (start, end) in enumerate(zip(epoch_start_nwb, epoch_end_nwb)):
+                nwbfile.add_epoch(start_time=start, stop_time=end, label='Epoch'+str(j+1))
+
+            trial_starts = [trial.start + epoch_start
+                            for epoch, epoch_start in zip(matin['log']['block'], epoch_start_nwb)
+                            for trial in epoch.trial]
+            trial_durations = [trial.duration for epoch in matin['log']['block'] for trial in epoch.trial]
+            trial_ends = [start_time + duration for start_time, duration in zip(trial_starts, trial_durations)]
             for k in range(len(trial_starts)):
                 nwbfile.add_trial(start_time=trial_starts[k], stop_time=trial_ends[k])
 
             # Processed position
-            pos_obj = Position(name='_position')
+            pos_obj = Position(name="PositionSeries")
```

Comment:
```suggestion
            pos_obj = Position(name="Position")
```
"Series" is reserved for objects that inherit from `TimeSeries`